main.bicep

```diff
@@ -366,12 +475,28 @@ resource sharedLogicAppIdentity 'Microsoft.ManagedIdentity/userAssignedIdentitie
   location: resourcesLocation
 }
 
+resource anomalyDetector 'Microsoft.CognitiveServices/accounts@2022-12-01' = if (deployAnomalyDetectionResources) {
+  name: 'msdyn-iiot-sdi-anomaly-detector-${uniqueIdentifier}'
+  location: resourcesLocation
+  sku: {
+    name: 'F0'
```

Comment:
Should probably add a note to `EXTENDED_USE.md` indicating when or why customers would want to modify this SKU?

Reply:
Added comment to EXTENDED_USE.md